docs/runtime/dotnet-metrics.md

```diff
@@ -4,7 +4,7 @@ linkTitle: .NET
 
 # Semantic Conventions for .NET Common Language Runtime (CLR) Metrics
 
-**Status**: [Experimental][DocumentStatus]
+**Status**: [Stable][DocumentStatus]
```

Comment:
If we decide to keep `dotnet.process.cpu.time` experimental in this PR, we should change it to 
```suggestion
**Status**: [Stable][DocumentStatus], Unless otherwise specified.
```